src/corelibs/U2Gui/src/util/InputWidgetsControllers.cpp

```diff
@@ -241,7 +241,7 @@ void ComboBoxController::addParameterToCmdLineSettings(QStringList &settings) {
     }
     settings << cmdLinePrefix;
     int curIndex = inputWidget->currentIndex();
-    if (parameters.size() > curIndex) {
+    if (curIndex >= 0 && parameters.size() > curIndex) {
         settings << parameters.at(curIndex);
     } else {
         settings << inputWidget->currentText();
```

Comment:
If `curIndex < 0`, an empty string will be added to the `settings`. Sometimes, this is bad behavior: an external tool will not work. Should I add some log message about this?

Reply:
Maybe, it'll be better to avoid this situation at all?

Reply:
Yes in this case some algorithm should be pre-selected by default and an invalid index value should be processed as an error.